### ISSUE TITLE
Copy to clipboard

### ISSUE THREAD

**Issue by usMn-Codr** [Fabi019/hid-barcode-scanner#303]
Can you add an option to copy the scanned value to the clipboard?
This would be very useful as we can send data to our PC using the phone link shared clipboard feature without needing bluetooth.

**Comment by WinLin97** [comment]
I see that in the newest version it's available by hand. 
You can Simple tap on text from barcode and that's all.

**Comment by usMn-Codr** [comment]
Thanks, I didn't notice you could do that. It would be nice if there was a toggleable option to automatically copy to clipboard as soon as the barcode is scanned.

**Comment by Fabi019** [comment]
Hi, currently it is possible to copy the value by manually tapping on it in the scanner after detected. I left the issue open because the suggested option to automatically copy the value after detection doesn't exist yet

**Comment by Fabi019** [comment]
Yes sure 👍 Ideally there should be a new preference in the [Settings.kt](https://github.com/Fabi019/hid-barcode-scanner/blob/main/app%2Fsrc%2Fmain%2Fjava%2Fdev%2Ffabik%2Fbluetoothhid%2FSettings.kt#L382) to enable the feature (Simmilar to how the vibration or sound is then implemented in the [Scanner.kt](https://github.com/Fabi019/hid-barcode-scanner/blob/main/app%2Fsrc%2Fmain%2Fjava%2Fdev%2Ffabik%2Fbluetoothhid%2FScanner.kt#L338))
